src/components/BestItemList.jsx

```diff
@@ -0,0 +1,64 @@
+import { useEffect, useState, useCallback } from "react";
+import { getItems } from "../apis/api";
+import styles from "./BestItemList.module.css";
+import ItemCard from "./ItemCard";
+import useAsync from "../hooks/useAsync";
+import ItemCardSkeleton from "../ui/Skeletons/ItemCardSkeleton";
+
+const ORDER_BY = "favorite";
+
+const BestItemList = ({ pageSize, title }) => {
+  const [items, setItems] = useState([]);
+  const [isLoading, loadingError, getItemsAsync] = useAsync(getItems);
+
+  const handleLoad = useCallback(
+    async (options) => {
+      const result = await getItemsAsync(options);
+      if (!result) return;
+
+      const { list } = result;
+      setItems(list);
+    },
+    [getItemsAsync]
+  );
+
+  useEffect(() => {
+    handleLoad({ pageSize: pageSize, orderBy: ORDER_BY });
```

Comment:
```suggestion
    handleLoad({ pageSize, orderBy: ORDER_BY });
```

위와 같이 [단축 속성명](https://github.com/codeit-bootcamp-frontend/16-Sprint-Mission/pull/160)을 사용하셔도될것같아요!